### PR TITLE
Docs-4279 Update Angelfish and Bluefin Release Notes Known Issues

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit updates both the Angelfish  22.02.4 and Bluefin 22.12-BETA.1 release notes Known Issues. Added the issue about upgrading from 22.02.4 to 22.12-BETA.1 and the workaround.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALE22.12.md [hunk 1]
```diff
@@ -450,7 +450,8 @@ Additional feature in future Bluefin releases:
 
 | Seen In | Key | Summary | Workaround | Resolved In |
 |---------|-----|---------|------------|-------------|
-| 22.12-BETA.1 | <a href="https://ixsystems.atlassian.net/browse/NAS-117940" target="_blank">NAS-117940</a> | Implements temporary fix for the return from `glfs_open()` to honor `O_DIRECTORY` flag | Pertains to an internal issue in Samba. After gluserfs is fixed with a permanent solution to this issue happens, this temporary fix will be reverted. | Targeted 22.12 |
+| 22.12-BETA.1  |  | Upgrading from 22.02.4 to 22.12-BETA.1 is unavailable. | Workaround is to either upgrade from a version before 22.01.4 or to upgrade to 22.12-BETA.2. | 2.12-BETA.2 |
```
Comment (william-gr) [inline content/SCALE/SCALE22.12.md:453]: unavailable is probably the wrong word. Maybe known to not work or cause issues? @Mrt134 

Also it is 22.02.4, not 22.01.4
And 22.12-BETA.2 not 2.12-BETA.2

Comment (micjohnson777) [reply content/SCALE/SCALE22.12.md:453]: Corrected typos and changed wording to "known to not work". I'll push this fix now.

File: content/SCALE/SCALE22.12.md [hunk 2]
```diff
@@ -450,7 +450,8 @@ Additional feature in future Bluefin releases:
 
 | Seen In | Key | Summary | Workaround | Resolved In |
 |---------|-----|---------|------------|-------------|
-| 22.12-BETA.1 | <a href="https://ixsystems.atlassian.net/browse/NAS-117940" target="_blank">NAS-117940</a> | Implements temporary fix for the return from `glfs_open()` to honor `O_DIRECTORY` flag | Pertains to an internal issue in Samba. After gluserfs is fixed with a permanent solution to this issue happens, this temporary fix will be reverted. | Targeted 22.12 |
+| 22.12-BETA.1  |  | Upgrading from 22.02.4 to 22.12-BETA.1 is known to not work. | Workaround is to either upgrade from a version before 22.02.4 or to upgrade to 22.12-BETA.2. | 22.12-BETA.2 |
```
Comment (william-gr) [inline content/SCALE/SCALE22.12.md:453]: `upgrade to 22.12-BETA.2`, maybe note that BETA.2 is not out yet... "wait to upgrade to 22.12-BETA.2 once it is released" or something

Comment (micjohnson777) [reply content/SCALE/SCALE22.12.md:453]: I'm in the BETA.2 rel note process now so I'll make sure this is in that as well as in the PR. I'll fix this to targeted for the fixed in.

**Review by tonyriv3 — APPROVED** [review]

**Pull request merged**